src/app/app-routing.module.ts

```diff
@@ -3,7 +3,7 @@ import { NgModule } from '@angular/core';
 import { MainCoreLegacyComponent } from '@components/main-core-legacy/main-core-legacy-component';
 import { MainComponentLegacyComponent } from '@components/main-legacy/main-legacy.component';
 import { LoginPageComponent } from '@components/login/login-page-component';
```

Comment:
https://github.com/dotCMS/core-web/blob/issue-16167-redesing-login-page/src/app/view/components/login/dot-login-component/login-container.component.ts

Also need work:

1. We need a `take(1)` [here](https://github.com/dotCMS/core-web/blob/issue-16167-redesing-login-page/src/app/view/components/login/dot-login-component/login-container.component.ts)

2. I think [this](https://github.com/dotCMS/core-web/blob/issue-16167-redesing-login-page/src/app/view/components/login/dot-login-component/login-container.component.ts#L40) should be in the `ngOnInit`